liberty-ls/src/main/java/io/openliberty/tools/langserver/utils/PropertiesValidationResult.java

```diff
@@ -0,0 +1,65 @@
+/*******************************************************************************
+* Copyright (c) 2022 IBM Corporation and others.
+*
+* This program and the accompanying materials are made available under the
+* terms of the Eclipse Public License v. 2.0 which is available at
+* http://www.eclipse.org/legal/epl-2.0.
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Contributors:
+*     IBM Corporation
+*******************************************************************************/
+package io.openliberty.tools.langserver.utils;
+
+import java.util.List;
+import java.util.logging.Logger;
+
+import io.openliberty.tools.langserver.ls.LibertyTextDocument;
+import io.openliberty.tools.langserver.model.propertiesfile.PropertiesEntryInstance;
+
+public class PropertiesValidationResult {
+    Integer lineNumber;
+    boolean hasErrors;
+    PropertiesEntryInstance entry;
+    LibertyTextDocument textDocumentItem;
+    
+    private static final Logger LOGGER = Logger.getLogger(PropertiesValidationResult.class.getName());
+
+    public PropertiesValidationResult(PropertiesEntryInstance entry) {
+        this.entry = entry;
+        this.textDocumentItem = entry.getTextDocument();
+        this.hasErrors = false;
+    }
+
+    public void validateServerProperty() {
+        if (entry.isComment() || !ServerPropertyValues.canValidateKeyValues(textDocumentItem, entry.getKey())) {
+            return;
+        }
+        List<String> validValues = ServerPropertyValues.getValidValues(textDocumentItem, entry.getKey());
+        if (!validValues.stream().anyMatch(entry.getValue()::equalsIgnoreCase)) { // check if validValues contains the property value, ignoring case
```

Comment:
Does Liberty ignore the case for all the pre-defined values for the various properties/env variables?

Reply:
We are following up on this. Apparently some props/vars are case sensitive, and others are not. Once we have identified all the recognized values, the code can be updated for this PR.

Reply:
Updated the code to follow the current state of case-sensitivity for the properties/variables.
In example: 
![image](https://user-images.githubusercontent.com/689163/186474960-e159ad8d-d1ef-45a5-98bf-05fbf4230b07.png)

Reply:
Should the test case be tweaked to cover case sensitive vs not?

Reply:
Currently the language server portion of the project does not have any tests. I can look into setting up a testing framework for the project next.

Reply:
k...that is a whole we will want to fill. Go ahead and open an issue to follow up on adding tests for the LS.